scheduler/scheduler.go

```diff
@@ -189,14 +189,13 @@ func (s *Scheduler) Sync(ctx context.Context, client schema.ClientMeta, tables s
 
 	// send migrate messages first
 	for _, tableOriginal := range tables.FlattenTables() {
-		// var table *schema.Table
-		table := tableOriginal.Copy(nil)
-		if syncClient.deterministicCQID {
-			table = schema.CqIDAsPK(table)
+		migrateMessage := &message.SyncMigrateTable{
+			Table: tableOriginal,
 		}
-		res <- &message.SyncMigrateTable{
-			Table: table,
+		if syncClient.deterministicCQID {
+			migrateMessage.Table = schema.CqIDAsPK(tableOriginal.Copy(nil))
```

Comment:
```suggestion
			migrateMessage.Table = schema.CqIDAsPK(tableOriginal)
```

Shouldn't the copy inside `CqIDAsPK` be enough?

Reply:
I think you reviewed an older version of the PR, Can you look at the latest version and see if you think this comment still applies?